.github/workflows/build_and_test.yml

```diff
@@ -199,6 +199,23 @@ jobs:
       build-tools-image: ${{ needs.build-build-tools-image.outputs.image }}-bookworm
     secrets: inherit
 
+  validate-compute-manifest:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install jsonschema
+        run: npm install -g @sourcemeta/jsonschema
+
+      - name: Validate manifest against schema
+        run: jsonschema validate -d https://json-schema.org/draft/2020-12/schema compute/manifest.schema.json compute/manifest.yaml 
```

Comment:
I think it would make sense to move this to a make command, to make it easier to run locally.
We do similar thing with jsonnet, see  https://github.com/neondatabase/neon/blob/3fd5a94a853604d26ff69acaf88c35f765e51698/.github/workflows/build_and_test.yml#L120-L123 and https://github.com/neondatabase/neon/blob/3fd5a94a853604d26ff69acaf88c35f765e51698/compute/Makefile#L44-L47